recipes/openbabel/bld.bat

```diff
@@ -0,0 +1,18 @@
+cmake ^
+      -G "%CMAKE_GENERATOR%" ^
+      -DCMAKE_INSTALL_PREFIX=%PREFIX% ^
```

Comment:
I wonder if this line is changed to `-DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^` then at least the `xcopy %PREFIX%\bin %PREFIX%\Library\bin /e /c` should not be needed?

Adding `-DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^` may also help.